web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -120,10 +133,67 @@
         href: "/dashboard",
         isAnchor: true,
       }}
-      nextButton={{ disabled: isNextButtonDisabled }}
+      nextButton={{
+        action: () => {
+          activeStep = 1;
+        },
+        disabled: !addressValidationResult.isValid,
+      }}
     >
       <div in:fade|global class="operation__send">
-        <ContractStatusesList items={statuses} />
+        <div class="operation__send-amount operation__space-between">
+          <p>Enter address:</p>
+          <Button
+            disabled={!scanner}
+            size="small"
+            on:click={() => {
+              scanQrComponent.startScan();
+            }}
+            text="SCAN QR"
+          />
+        </div>
+        <Textbox
+          className="operation__send-address
+          {!addressValidationResult.isValid
+            ? 'operation__send-address--invalid'
+            : ''}"
+          type="multiline"
+          bind:value={address}
+        />
+        <ScanQR
+          bind:this={scanQrComponent}
+          bind:scanner
+          on:scan={(event) => {
+            address = event.detail;
+          }}
+        />
+      </div>
+    </WizardStep>
+    <WizardStep
+      step={1}
+      {key}
+      backButton={{
+        action: () => {
+          activeStep = 0;
+        },
+      }}
+      nextButton={{
+        action: () => {
+          activeStep = 2;
+        },
+        disabled: isNextButtonDisabled,
+      }}
+    >
+      <div in:fade|global class="operation__send">
+        <ContractStatusesList items={statuses}>
+          <AppAnchorButton
+            className="allocate-button"
+            href="/dashboard/allocate"
+            text="Shield more DUSK"
+            variant="tertiary"
+          />
```

Comment:
I would personally increase the margin a bit. Something like this:

<img width="585" alt="Screenshot 2024-09-19 at 16 19 54" src="https://github.com/user-attachments/assets/afeac2f0-1b9e-4d92-bce4-03a63f0046bc">

which looks more like the design 

<img width="640" alt="Screenshot 2024-09-19 at 1 03 35" src="https://github.com/user-attachments/assets/dcdbd096-8e01-4d10-ae91-4ae3ec5a78f4">

compared to the current implementation:

<img width="582" alt="Screenshot 2024-09-19 at 1 01 51" src="https://github.com/user-attachments/assets/203f87a5-fd5a-4533-9db3-b911f502b6e1">

Reply:
The problem here is that the Figma specifies a gap value of `0.5em`, which is the same that is used in the implementation.  What is different is that the _label_ has a `line-height` of `150%`, which I will add to the component now.

We need to talk with @laremas about this because the plan is for the design token values to be derived directly from Figma and we shouldn't use _eyeballed_ values that "look more like the design".  It should just be the value that's specified.